docs/topics/gradle.md

```diff
@@ -258,33 +258,27 @@ kotlin.stdlib.default.dependency=false
 
 The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
 
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+Add the dependency `kotlin-test` to the `commonTest` source set, and the Gradle plugin will infer the corresponding 
+test dependencies for each source set:
+* `kotlin-test-common` and `kotlin-test-annotations-common` for common source sets
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets
```

Comment:
Please avoid using `/` as a conjunction. It's often unclear what you're replacing with it: and, or, something else?
I'd mention only `kotlin-test-junit` in the first sentence because it is actually inferred with `kotlin-test`. Perhaps we can add a second sentence about JUnit 5  and TestNG.

Reply:
Should I also do sth with ```/``` in [what's new](https://github.com/JetBrains/kotlin-web-site/pull/2192/files#r607146909)?